### PR TITLE
changed table format from html to markdown in content/blog/entries/....

### LINKED ISSUE #248: Convert HTML tables to Markdown tables
## Overview
Now that we have tables styled by default in [Add default styling for tables by sp35 · Pull Request #227 · creativecommons/creativecommons.github.io-source](https://github.com/creativecommons/creativecommons.github.io-source/pull/227), we should replace instances of HTML tables with tables formatted in Markdown wherever possible. It will make things more readable.

## Notes on PRs
Feel free to send PRs for one table at a time. Please make sure to include before and after screenshots in your PRs.

## Files with HTML Tables
- [ ] [`content/blog/entries/cc-vocabulary-the-main-course/contents.lr`](https://github.com/creativecommons/creativecommons.github.io-source/blob/master/content/blog/entries/cc-vocabulary-the-main-course/contents.lr)
- [ ] [`content/community/contents.lr`](https://github.com/creativecommons/creativecommons.github.io-source/blob/master/content/community/contents.lr)
- [x] [`content/internships/contents.lr`](https://github.com/creativecommons/creativecommons.github.io-source/blob/master/content/internships/contents.lr)

(Be sure to check  for activity below)

### PR DESCRIPTION
**Pull request by ayu023ban**
## Fixes

Fixes #248 by @kgodey 

## Description

changed table of /content/blog from html format to markdown

## Screenshots

before:
Screenshot_2020-06-08 Convert HTML tables to Markdown tables · Issue #248 · creativecommons creativecommons github io-source

after:

Screenshot_2020-06-08 ayu023ban creativecommons github io-source

code:
Screenshot from 2020-06-08 01-47-15

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 1]
```diff
@@ -1,26 +1,28 @@
-title: CC Vocabulary: the main course
----
+## title: CC Vocabulary: the main course
+
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:2]: We use the `---` notation in our other blogs. There's no need to change it.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
----
-pub_date: 2019-07-19
----
+
+## author: dhruvkb
+
+## series: gsoc-2019-vocabulary
+
+## pub_date: 2019-07-19
+
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:17]: ^ Same.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
 
 A little over seven weeks ago we initialised CC Vocabulary. It was a simple recipe, a set of components for other apps to
 use for consistency and ease of development.
 
-- **Week 1:**    we assembled the ingredients: _Vue CLI, Vue Styleguidist, Theo, Lodash, Font Awesome_.
-- **Week 2:**    we cleaned up the kitchen: _Dockerised our project and set up CI/CD_.
-- **Week 3:**    we ground spices: _developed and incorporated design tokens_.
+- **Week 1:** we assembled the ingredients: _Vue CLI, Vue Styleguidist, Theo, Lodash, Font Awesome_.
+- **Week 2:** we cleaned up the kitchen: _Dockerised our project and set up CI/CD_.
+- **Week 3:** we ground spices: _developed and incorporated design tokens_.
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:25]: This was not necessary, but ok.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 4]
```diff
@@ -50,105 +52,32 @@ brainchild comes of age.
...
-    </tr>
-</tbody>
-</table>
+|                    | Tokens                      |
+| ------------------ | :-------------------------- |
+| Colors             | ![colors](colors.png)       |
+| Fonts              | ![fonts](fonts.png)         |
+| Spaces             | ![spaces](spaces.png)       |
+|                    | **Elements**                |
+| Button             | ![button](button.png)       |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:61]: Why does this image have `button`...

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 5]
```diff
@@ -50,105 +52,32 @@ brainchild comes of age.
...
+| InputField         | ![](inputfield.png)         |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:62]: ...when this image just has ``?

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 6]
```diff
@@ -50,105 +52,32 @@ brainchild comes of age.
...
-        <th colspan="2"><h3>Templates</h3></th>
-    </tr>
-    <tr>
-        <td>Index</td>
-        <td><img src="orange.png" onclick="this.src = 'green.png'"></td>
-    </tr>
-</tbody>
-</table>
+|                    | Tokens                      |
+| ------------------ | :-------------------------- |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:56]: The token heading is on the same level as `**Elements**`, `**Layouts**` and `**Patterns**` below so making it the table title isn't right.
```suggestion
| Component          | Image                       |
| ------------------ | :-------------------------- |
| **Elements**       |                             |
```

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 7]
```diff
@@ -50,105 +52,32 @@ brainchild comes of age.
...
+| Colors             | ![colors](colors.png)       |
+| Fonts              | ![fonts](fonts.png)         |
+| Spaces             | ![spaces](spaces.png)       |
+|                    | **Elements**                |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:60]: Preferably move this subheading (and those below) into the left cell.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr (hunk 4 above)
Comment (ayu023ban) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:61]: i provided alternate value if the image doesnot load.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr (hunk 7 above)
Comment (ayu023ban) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:60]: ok

Comment (ayu023ban) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:60]: should i remove the alternate values to the image or add alternate values to other rows as well?

Comment (dhruvkb) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:60]: Add alternate values. They're good for a11y.

**Comment by ayu023ban** [comment]
Made requested changes.
You can check.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr (hunk 1 above)
Comment (dhruvkb) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:2]: @ayu023ban you didn't revert this.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr (hunk 2 above)
Comment (dhruvkb) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:17]: @ayu023ban you didn't revert this.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 8]
```diff
@@ -50,105 +52,33 @@ brainchild comes of age.
...
+| Container          | ![container_image](container.png)                    |
+| Grid               | ![grid_image](grid.png)                              |
+| Table              | ![table_image](table.png)                            |
+| **Patterns**       |                                                      |
+| Header             | ![header_image](header.png)                          |
+| Footer             | ![footer_image](footer.png)                          |
+| Locale             | ![locale_image](locale.png)                          |
+| Hello              | ![hello_image](hello.png)                            |
+| **Templates**      |                                                      |
+| Index              | ![](orange.png)                                      |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:81]: Missed one.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 9]
```diff
@@ -50,105 +52,33 @@ brainchild comes of age.
...
+| Fonts              | ![fonts_image](fonts.png)                            |
+| Spaces             | ![spaces_image](spaces.png)                          |
+| **Elements**       |                                                      |
+| Button             | ![buttons_image](button.png)                         |
+| InputField         | ![inputfield_image](inputfield.png)                  |
+| SelectField        | ![select_field_image](selectfield.png)               |
+| Heading            | ![heading_image](heading.png)                        |
+| Paragraph          | ![paragraph_image](paragraph.png)                    |
+| LicenseBadge       | ![licensebadge_image](licensebadge.png)              |
+| LicenseIconography | ![license_iconography_image](licenseiconography.png) |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:68]: For consistency with the other alternative text.
```suggestion
| LicenseIconography | ![licenseiconography_image](licenseiconography.png)  |
```

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 10]
```diff
@@ -50,105 +52,33 @@ brainchild comes of age.
...
-</tbody>
-</table>
+|                    |                                                      |
+| ------------------ | ---------------------------------------------------- |
+| **Tokens**         |                                                      |
+| Colors             | ![colors_image](colors.png)                          |
+| Fonts              | ![fonts_image](fonts.png)                            |
+| Spaces             | ![spaces_image](spaces.png)                          |
+| **Elements**       |                                                      |
+| Button             | ![buttons_image](button.png)                         |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:62]: For consistency with the other alternative text.
```suggestion
| Button             | ![button_image](button.png)                          |
```

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 11]
```diff
@@ -50,105 +52,33 @@ brainchild comes of age.
...
+| InputField         | ![inputfield_image](inputfield.png)                  |
+| SelectField        | ![select_field_image](selectfield.png)               |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:64]: For consistency with the other alternative text.
```suggestion
| SelectField        | ![selectfield_image](selectfield.png)                |
```

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr [hunk 12]
```diff
@@ -50,105 +52,33 @@ brainchild comes of age.
...
-    <tr>
-        <th colspan="2"><h3>Templates</h3></th>
-    </tr>
-    <tr>
-        <td>Index</td>
-        <td><img src="orange.png" onclick="this.src = 'green.png'"></td>
-    </tr>
-</tbody>
-</table>
+|                    |                                                      |
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-the-main-course/contents.lr:55]: Oh and you missed the titles in the table. See https://github.com/creativecommons/creativecommons.github.io-source/pull/293#discussion_r436456195.

Comment (ayu023ban) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:55]: i can't find any way to add multiple headers in markdown table so can you provide any?

Comment (ayu023ban) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:55]: I can make 5 different tables with corresponding headers

Comment (ayu023ban) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:55]: the code will look like this:


|        | **Tokens** |                             |
| ------ | :--------: | --------------------------- |
| Colors |            | colors_image |
| Fonts  |            | fonts_image   |
| Spaces |            | spaces_image |

|                    | **Elements** |                                                     |
| ------------------ | :----------: | --------------------------------------------------- |
| Button             |              | button_image                         |
| InputField         |              | inputfield_image                 |
| SelectField        |              | selectfield_image               |
| Heading            |              | heading_image                       |
| Paragraph          |              | paragraph_image                   |
| LicenseBadge       |              | licensebadge_image             |
| LicenseIconography |              | licenseiconography_image |
| ProgressBar        |              | progressbar_image               |
| Shield             |              | shield_image                         |

|           | **Layouts** |                                   |
| --------- | :---------: | --------------------------------- |
| Container |             | container_image |
| Grid      |             | grid_image           |
| Table     |             | table_image         |

|        | **Patterns** |                             |
| ------ | :----------: | --------------------------- |
| Header |              | header_image |
| Footer |              | footer_image |
| Locale |              | locale_image |
| Hello  |              | hello_image   |

|       | **Templates** |                             |
| ----- | :-----------: | --------------------------- |
| Index |               | orange_image |



i have made three columns to put title in center.
if we want to put title in left then there is no need of middle column

**Comment by kgodey** [comment]
I'll let @dhruvkb review/merge this.

File: content/blog/entries/cc-vocabulary-the-main-course/contents.lr (hunk 12 above)
Comment (dhruvkb) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:55]: Don't split this into five tables, just put this as the header at the top of the table:
```
| Component          | Image                       |
| ------------------ | :-------------------------- |
```
Component is the heading that covers all subheadings, from tokens to templates.

Comment (dhruvkb) [reply content/blog/entries/cc-vocabulary-the-main-course/contents.lr:55]: Don't split this into five tables, just put this as the header at the top of the table:
```
| Component          | Image                       |
| ------------------ | :-------------------------- |
```
Component is the heading that covers all subheadings, from tokens to templates.

**Pull request merged**